content/vendor/guides/helm-chart.md

```diff
@@ -225,12 +224,18 @@ data:
 ...
 ```
 
+Next, we can make a release and get ready to test it out
+
+```shell
+make release
+```
+
 Deploy the new release
 ![Helm Chart Check for Updates 2](/images/guides/kots/helm-chart-check-for-updates-2.png)
 
 Update the User and Password fields in kotsadm `Config` and verify the `grafana` secret got updated.
 
-```shell
+```text
```

Comment:
this is where I stopped reading, but I might have a few more tweaks -- but wanted to get this lined up so we don't have conflicts.